Lavinia-api/Extensions/ServicesExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AspNetCoreRateLimit;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LaviniaApi.Extensions
+{
+    public static class ServicesExtensions
+    {
+
+        public static void AddRateLimiting(this IServiceCollection services, IConfiguration Configuration)
```

Comment:
Shouldn't this return an IServiceCollection? Otherwise, how could you chain this?